lldb/docs/use/mcp.md

```diff
@@ -0,0 +1,68 @@
+# Model Context Protocol (MCP)
+
+LLDB supports for the [Model Context Protocol][1] (MCP). This structured,
+machine-friendly protocol allows AI models to access and interact with external
+tools, such as the debugger. Using MCP, an AI agent can execute LLDB commands
+to control the debugger: set breakpoints, inspect memory, step through code.
+This can range from helping you run a specific command you can't immediately
+remember to a fully agent-driven debugging experiences
+
+## MCP Server
+
+To start the MCP server in LLDB, use the `protocol-server start` command.
+Specify `MCP` as the protocol and provide a URI to listen on. For example, to
+start listening for local TCP connections on port `59999`, use the following
+command:
+
+```
+(lldb) protocol-server start MCP listen://localhost:59999
+MCP server started with connection listeners: connection://[::1]:59999, connection://[127.0.0.1]:59999
+```
+
+The server will automatically stop when exiting LLDB, or it can be stopped
+explicitly with the `protocol-server stop` command.
```

Comment:
This means you still use the CLI while the server is active?

....does this mean you could run a command and ask the model what it thought of the output? That sounds cool.

Reply:
Also, how do you check if it is active? `protocol-server` start again and it errors?

Reply:
> ....does this mean you could run a command and ask the model what it thought of the output?
Not right now, but that could be done using the "notification" feature in MCP. 

> Also, how do you check if it is active? protocol-server start again and it errors?
Correct

Reply:
And I think the workaround would be to say "run command X and tell me what you think the output means".